pyznap/process.py

```diff
@@ -65,6 +65,10 @@ def check_returncode(self):
                               HoldTagExistsError):
                     if reason == error.strerror:
                         raise error(dataset)
+        elif self.stderr.startswith('ssh:'):
```

Comment:
this should be included in the next commit...

Reply:
I found that ssh will always return its error message with "ssh:*", it could be a connection error or something else. Maybe we could use a more rubost criterion to test the connection error.
Anyway, when ssh report a error, the connection must be lost, this criterion works in my test, but may be improved.